src/gamecontroller/gamecontrollerset.h

```diff
@@ -38,21 +38,17 @@ class GameControllerSet : public SetJoystick
 
     virtual void refreshAxes();
 
-    virtual void readConfig(QXmlStreamReader *xml); // GameControllerSetXml
-    virtual void readJoystickConfig(QXmlStreamReader *xml, QHash<int, SDL_GameControllerButton> &buttons,
-                                    QHash<int, SDL_GameControllerAxis> &axes,
-                                    QList<SDL_GameControllerButtonBind> &hatButtons); // GameControllerSetXml class
+    virtual void readConfig(QXmlStreamReader *xml);
 
   protected:
     void populateSticksDPad();
 
   public slots:
     virtual void reset();
+    void applyHapticFeedback();
```

Comment:
I think a description of this method would be nice.

Reply:
Fixed in next patchset.